dietpi.txt

```diff
@@ -11,137 +11,137 @@
 
 ##### Networking Options #####
 
-#	If both Ethernet and Wifi are enabled, Wifi will take priority and Ethernet will be disabled.
-#		1=enabled
+#    If both Ethernet and Wifi are enabled, Wifi will take priority and Ethernet will be disabled.
+#        1=enabled
 AUTO_SETUP_NET_ETHERNET_ENABLED=1
 AUTO_SETUP_NET_WIFI_ENABLED=0
 
-#	If using WiFi, please edit the following to pre-enter creds /boot/dietpi-wifi.txt
+#    If using WiFi, please edit the following to pre-enter creds /boot/dietpi-wifi.txt
 
-#	Enter your Static Network details below, if applicable.
+#    Enter your Static Network details below, if applicable.
```

Comment:
Appears to be 4 spaces, instead of a single tab.

Ideally these should be tabs, however, as I use the 4 spaces for tab anyway, should have same affect.

@MichaIng 

I'am fine with this, I can always edit it afterwards if not as expected in viewer?

Reply:
@Fourdee 
Just checked, actually in `dietpi.txt` here on GitHub tabs are just translated into two spaces by default, although can be changed. Maybe GitHub defaults depend on file ending or what 😄.

We should really align GitHub and your local editor. Maybe once and for all change all DietPi files to <tab>=<tab> with 4 space width?
But nothing urgent, for me PR also okay as is 👍.